apps/req.c

```diff
@@ -769,21 +769,21 @@ int req_main(int argc, char **argv)
 
             /* Set up V3 context struct */
 
-            X509V3_set_ctx(&ext_ctx, x509ss, x509ss, NULL, NULL, 0);
+            X509V3_set_ctx(&ext_ctx, x509ss, x509ss, NULL, NULL, X509V3_CTX_REPLACE);
```

Comment:
Why this change is needed? Should that be somehow reflected in the commit message? Similarly down below where you also add this flag.

Reply:
See below...

Reply:
This turned out to be helpful with the approach I plan to submit on solving #13603
because there, when a user indicates "no" for an SKID or AKID, it will tentatively add an empty entry to track this,
and just before signing it will either remove the empty entry if present and else add a default entry if nothing is present yet.